corehq/apps/users/migrations/0072_remove_location_fields_in_user_data.py

```diff
@@ -0,0 +1,63 @@
+from django.db import migrations
+
+from couchdbkit import ResourceNotFound
+
+from corehq.apps.users.util import user_location_data
+from corehq.apps.users.models import CommCareUser
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def remove_keys_from_data(apps, schema_editor):
+    SQLUserData = apps.get_model('users', 'SQLUserData')
+
+    for user_data in SQLUserData.objects.all():
+        data = user_data.data
+        keys_to_remove = ['commcare_location_id', 'commcare_location_ids', 'commcare_primary_case_sharing_id']
+
+        modified = False
+        for key in keys_to_remove:
+            if key in data:
+                del data[key]
+                modified = True
+
+        if modified:
+            user_data.data = data
```

Comment:
nit: I think this reassignment is unnecessary since `user_data.data` is the same dict as `data`.
```suggestion
```